### DISCUSSION TITLE
Nexct cloud AIO Error accessing "The server is temporarily unable to service your request due to maintenance downtime or capacity problems"

### DISCUSSION THREAD

**Discussion by auceto** [nextcloud/all-in-one#5145] [Questions] [answered]
Hi,
I done a lot installation in the past whitout any issue but with this version, from scratch, I'm suffering a lot of problem.

I installed Nextcloud AIO from scratch following the guide, this is my configuration:

Nextcloud AIO v9.4.1
Nextcloud Hub 8 29.0.4
Operating System: Linux 6.6.31+rpt-rpi-2712 aarch64
CPU: Raspberry Pi 5 Model B Rev 1.0 (4 cores)
Memory: 7.86 GB
Server time: Sun Aug 18 16:46:20 CEST 2024

Now error 503 accessing at the web page and "ERROR: Connection disallowed: IP address '172.18.0.1' has been dropped" in the nextcloud log

I apply the commands showed in this thread:
[https://github.com/nextcloud/all-in-one/discussions/3654](https://github.com/nextcloud/all-in-one/discussions/3654)

but I can do this at every restart.

I follow the IPV6 instruction here:
[https://github.com/nextcloud/all-in-one/blob/main/docker-ipv6-support.md](https://github.com/nextcloud/all-in-one/blob/main/docker-ipv6-support.md)
and here:
[https://github.com/nextcloud/all-in-one/discussions/4989](https://github.com/nextcloud/all-in-one/discussions/4989)
to recreate the docker network without success.

is there some tips that I loss?

**Comment by szaimen** [comment]
Hi, can you post the output of `sudo docker info` here?

**Reply by auceto** [reply]
Hi,
Sorry but I format all another time for install it from scratch again.
if the problem occurs again this time I will definitely post the output of docker info
Thanks

**Reply by auceto** [reply]
I'll try to post the result of the docker info anyway, but consider that at the moment there is only the master container still to configure; I'm away from home and I can't do it
[docker-info.txt](https://github.com/user-attachments/files/16802851/docker-info.txt)

**Reply by auceto** [reply]
HI szaimen,
sorry for late
I tried reinstalling a bunch of times, even the latest beta version. Nothing to do every time I can't log in. Now once configured if I enter my dns the page says "the site can't provide a secure connection"
in the Nexcloud container logs this error appears
ERROR: Connection disallowed: IP address '172.18.0.1' has been dropped
I think that this is the real problem, but I don't know how to solve.

I found these instructions but now they don't work anymore:

sudo docker exec --user www-data -i nextcloud-aio-nextcloud sed -i -re 's/(allowed_clients = .*)/\1,172.18.0.1/' /usr/local/etc/php-fpm.d/www.conf
sudo docker exec --user root -i nextcloud-aio-nextcloud killall -USR2 php-fpm

now I'm stuck and I have no more ideas on what to do

thanks in advance

**Reply by szaimen** [reply]
Hm... Do you run a reverse proxy in front of AIO? Also, can you post the output of docker info again?

**Reply by auceto** [reply]
no reverse proxy
I used compose file in the last installation (with the docker run I've the same issue)

[Nextcloud AIO-no reverse.txt](https://github.com/user-attachments/files/16995657/Nextcloud.AIO-no.reverse.txt)


here docker info result
thank again
[dockerinfo.txt](https://github.com/user-attachments/files/16995628/dockerinfo.txt)

**Reply by szaimen** [reply]
> WARNING: No memory limit support
> WARNING: No swap limit support

How did you install docker?

**Reply by auceto** [reply]
My docker is installed on Raspberry by Openmediavault.
Previous versions of NC have always worked with this.

**Reply by auceto** [reply]
Now I solved WARNING for docker in my Raspbarry thanks to this
[How to Fix “No memory limit support” for Docker in Raspberry Pi](https://dalwar23.com/how-to-fix-no-memory-limit-support-for-docker-in-raspberry-pi/)
I reset and reinstalled NC AIO another time, but I've the same problems.
the NC page now show this

![image](https://github.com/user-attachments/assets/0f16d6c5-e937-43ab-ae5f-a191e0a8f042)

this the new docker info

[dockerinfo.txt](https://github.com/user-attachments/files/16997352/dockerinfo.txt)

**Reply by auceto** [reply]
**another update**
with the instruction published above:
sudo docker exec --user www-data -i nextcloud-aio-nextcloud sed -i -re 's/(allowed_clients = .*)/\1,172.18.0.1/' /usr/local/etc/php-fpm.d/www.conf
sudo docker exec --user root -i nextcloud-aio-nextcloud killall -USR2 php-fpm

I can enter on NC Instance but I've to do this at every reboot

**Reply by szaimen** [reply]
How exactly did you install docker? With the script mwntioned in the readme or different?

**Reply by auceto** [reply]
In omv, OpenMediaVault, docker is installed internally from interface. I don't know the script that they use.
If you have same dubt about this, I can do an installation on another Raspberry with fresh operative system installation

**Reply by szaimen** [reply]
Yes, I am wondering if OMV does something fancy to the docker installation as I cannot reproduce the problem on my test server running Ubuntu server LTS and the official docker install script... Also weird that you seem to be the only one that runs into this...

**Reply by auceto** [reply]
Ok
Give me some time and i report results asap

**Reply by auceto** [reply, marked as answer]
Hi szaimen,
as I promised you, I've repeat the installation on another Raspberry (Rappberry pi 4) with fresh installation and with official docker installation; the NC AIO installation is went without any issue.
So as you have suspected the problem is the OMV docker plug in.
Now I have to plan a strategy, because I don't want to keep two Raspies turned on.

**Reply by szaimen** [reply]
I see. Maybe you could open a thread in a OMV forum for help on the topic and post the link here?

**Reply by auceto** [reply]
Maybe, but I'm doing another attemps in my RPI 4:
to be sure that the problems seems to be OMV, now I install another OMV instance and then I install NC AIO.
If the problems will be the same, yes I'll try to open an issue on the OMV Forum

**Reply by szaimen** [reply]
All right. Makes sense

**Reply by auceto** [reply]
Nothing to do
With the new OMV installation NC AIO start without any problem.
To this point I think that the OMV installation on Rasberry 5 have some problema but I don't have the necessary expertise to solve it
so I'm at a crossroads:
a) I repeat the OMV installation from scratch on the Raspi5 
b) I get another Raspi (4) on for NC only

What would you advise me to do?

Thanks for support

**Reply by szaimen** [reply]
If you have enough time and not so much money, I would recommend option a. If you have less time and enough money, I would recommend option b. In case of option b, I would probably recommend ubuntu server 24.04 LTS as OS and then install docker using the recommended install script.

**Reply by auceto** [reply]
Hi szaimen,
at the end, I reinstalled OMV from scratch e then NC AIO 30.
Now I've no problem to access NC but in the log I found houndred error "Driver Exception".
I saw that there is a discussion here in github, but not real solution.
may this will be solved in next release?

**Reply by szaimen** [reply]
> Driver Exception

This is a bug in Nextcloud 30.0.0 and will be fixed in Nextcloud 30.0.1 afaik
